packages/xo-web/src/common/select-objects.js

```diff
@@ -358,10 +360,26 @@ export const SelectPool = makeStoreSelect(
 
 export const SelectSr = makeStoreSelect(
   () => {
-    const getSrsByContainer = createGetObjectsOfType('SR')
-      .filter((_, { predicate }) => predicate || isSrWritable)
-      .sort()
-      .groupBy('$container')
+    const getSrsByContainer = createSelector(
+      createGetObjectsOfType('SR')
+        .filter((_, { predicate }) => predicate || isSrWritable)
+        .sort(),
+      srs => {
+        for (const sr1 of srs) {
+          for (const sr2 of srs) {
```

Comment:
What about O(nlog(n)) instead of O(n²)?
```js
const { length } = srs
if (length < 2) {
  // return ...
}
for (let i = 1; i < srs.length; ++i) {
  for (let j = 0; j < i; ++j) {
    // ...
  }
}
```
Bonus: you can remove the `sr1.id !== sr2.id` test \o/